### ISSUE TITLE
Avoid HypothesisWarning when run in Docker

### ISSUE THREAD

**Issue by Stranger6667** [schemathesis/schemathesis#912]
```
/usr/local/lib/python3.8/site-packages/hypothesis/database.py:60: HypothesisWarning: The database setting is not configured, and the default location is unusable - falling back to an in-memory database for this session.  path='/app/.hypothesis/examples'
  warnings.warn(
```

Maybe we can set it up explicitly to use in-memory db or make this location writable. I am not sure about workflows when this data will persist, but I assume they are possible with, e.g., volumes.

**Comment by Stranger6667** [comment]
Closed via a61453e

**Comment by magnusvmt** [comment]
I still have this issue running the latest docker image.
